selfdrive/car/tests/routes.py

```diff
@@ -59,6 +59,7 @@
   CarTestRoute("f08912a233c1584f|2022-08-11--18-02-41", GM.BOLT_EUV, segment=1),
   CarTestRoute("555d4087cf86aa91|2022-12-02--12-15-07", GM.BOLT_EUV, segment=14),  # Bolt EV
   CarTestRoute("38aa7da107d5d252|2022-08-15--16-01-12", GM.SILVERADO),
+  CarTestRoute("5085c761395d1fe6|2023-04-07--10-32-23", GM.TRAILBLAZER),
```

Comment:
this route is missing the rlogs at the beginning of the route. can you replace this with another route that has rlogs for the whole time?

Reply:
Okay, I will find another one.

Reply:
I updated it.